src/PropertyInfo/TypeInfoParsers/NativeTypeInfoParser.php

```diff
@@ -19,11 +17,8 @@
  *
  * @author Mihai Stancu <stancu.t.mihai@gmail.com>
  */
-abstract class NativeTypeInfoParser implements TypeInfoParserInterface
+trait NativeTypeInfoParser
```

Comment:
IMO, as it's on unstable version and was not existent before, this could be break without issue.

Let @dunglas decide about it. ;-)